src/webview/android/mod.rs

```diff
@@ -228,6 +229,10 @@ impl InnerWebView {
 
   pub fn print(&self) {}
 
+  pub fn url(&self) -> Url {
+    todo!();
```

Comment:
this will panic, could it maybe replaced with a default value? or change the function signature to use `Result`

Reply:
My assumption was that we actually want this function on android too, I just don't know how to implement it that's why I marked it as `todo`.

Or is this not possible on android?

Reply:
No it is possible, let me add it